components/Registry.tsx

```diff
@@ -172,7 +173,12 @@ const Registry = () => {
           {name}
           {version && `@${version}`} - deno.land/x
         </title>
-        <meta name="description" content="A third party module for Deno." />
+        {metaDescription({
+          title: "deno.land/x",
+          description: "A third party module for Deno.",
+          url: "https://deno.land/v1/",
```

Comment:
I think this component is associated with many URLs, none of which are https://deno.land/v1/

Reply:
fixed 🙂